ChatApp/Janus/Janus.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  File.swift
+//  ChatApp
+//
+//  Created by Kalvin Dela Raga on 1/27/21.
+//
+
+import UIKit
+import Firebase
+
+class Janus {
+    
+    let window: UIWindow
+    
+    private var listener: AuthStateDidChangeListenerHandle?
+    
+    init(window: UIWindow) {
+        self.window = window
+    }
+    
+    func start() {
+        listener = Auth.auth().addStateDidChangeListener { (auth, user) in
```

Comment:
Apply DRY, this Auth.auth() can be renamed

Reply:
This is from Firebase API. See [here](https://firebase.google.com/docs/reference/swift/firebaseauth/api/reference/Classes/Auth#/c:objc(cs)FIRAuth(cm)auth)